### PR TITLE
vulkaninfo: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/vulkaninfo.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# vulkaninfo
+
+> Display system Vulkan information.
+> More information: <https://vulkan.lunarg.com/doc/view/latest/windows/vulkaninfo.html>.
```
Comment (spageektti) [inline pages/linux/vulkaninfo.md:4]:
```suggestion
> More information: <https://vulkan.lunarg.com/doc/view/latest/linux/vulkaninfo.html>.
```

**Comment by spageektti** [comment]
Does it work only for Linux because it has documentation for Linux, Windows, and iOS?

**Comment by Managor** [comment]
Most probably. I tend to always put new commands under Linux because that's the platform I verify it on.

**Review by spageektti — APPROVED** [review]

**Comment by fazlearefin** [comment]
This should be moved to the `common` folder as it is not solely meant for linux.

As for the `More information` doc link, the following 3 links all point to the same page:
- https://vulkan.lunarg.com/doc/view/latest/mac/vulkaninfo.html
- https://vulkan.lunarg.com/doc/view/latest/linux/vulkaninfo.html
- https://vulkan.lunarg.com/doc/view/latest/windows/vulkaninfo.html

File: pages/linux/vulkaninfo.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> Display system Vulkan information.
+> More information: <https://vulkan.lunarg.com/doc/view/latest/linux/vulkaninfo.html>.
+
+- Print full Vulkan information:
+
+`vulkaninfo`
+
+- Print a summary:
+
+`vulkaninfo --summary`
```
Comment (CleanMachine1) [inline pages/linux/vulkaninfo.md:12]:
```suggestion
`vulkaninfo --summary`

- Make a HTML document of the full Vulkan information:

`vulkaninfo --html`
```

**Pull request merged**